installing/installing_vsphere/ipi/installing-vsphere-installer-provisioned-customizations.adoc

```diff
@@ -56,6 +56,13 @@ include::modules/installation-configure-proxy.adoc[leveloffset=+2]
 
 include::modules/configuring-vsphere-regions-zones.adoc[leveloffset=+2]
 
+// Services for an external load balancer
+include::modules/nw-osp-services-external-load-balancer.adoc[leveloffset=+1]
+
+// Configuring an external load balancer
```

Comment:
Does this need to say "Configuring a user-managed load balancer" instead, so it's consistent throughout?